src/definition_store.cpp

```diff
@@ -142,14 +142,14 @@ bool DefinitionStore::hasAbstractNode() {
   return false;
 }
 
-size_t DefinitionStore::verifyAbstractParent(const size_t sysNodeName) {
+bool DefinitionStore::verifyAbstractParent(const size_t sysNodeName) {
```

Comment:
`verifyAbstractParent` returns true what does it mean? So your method name should also fit to the return!
e.g. `isEmpty`